api/src/api/admin.js

```diff
@@ -181,4 +182,37 @@ router.delete(
   }),
 );
 
+// Create a translation object
+router.post(
+  '/translation',
+  errorWrap(async (req, res) => {
+    const newTranslation = new Translation(req.body);
+    await newTranslation.save();
+    res.json({
+      code: 201,
+      message: `Succesfully created new Translation object`,
+      success: true,
+      result: newTranslation,
+    });
+  }),
+);
+
+// Add a translation message
+router.put(
+  '/translation',
+  errorWrap(async (req, res) => {
+    const updatedTranslation = await Translation.findOne({
+      language: { $eq: req.body.language },
+    });
+    updatedTranslation.messages.set(req.body.key, req.body.message);
+    await updatedTranslation.save();
+    res.json({
+      code: 200,
+      message: `Successfully added ${req.body.language} translation for ${req.body.key}`,
+      success: true,
+      result: updatedTranslation,
+    });
+  }),
+);
```

Comment:
One random nit here - you can get rid of the repeated `req.body...` code by doing something like this at the beginning
```
const { language, key, message} = req.body;
```